source/replication/server-side-replication-resynchronize-remote.rst

```diff
@@ -0,0 +1,300 @@
+        for a remote target to synchronize data into *any* bucket in the MinIO
+        cluster. To restrict the policy to specific buckets, specify those 
+        buckets as an element in the ``Resource`` array similar to 
+        ``"arn:aws:s3:::bucketName/*"``.
+
+      Use the :mc-cmd:`mc admin policy add` to add this policy to *both*
+      deployments. You can then create a user on both deployments using
+      :mc-cmd:`mc admin user add` and associate the policy to those users
+      with :mc-cmd:`mc admin policy set`.
+
+MinIO strongly recommends creating users specifically for supporting 
+bucket replication operations. See 
+:mc:`mc admin user` and :mc:`mc admin policy` for more complete
+documentation on adding users and policies to a MinIO cluster.
+
+Existing Object Replication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Resynchronization requires :ref:`existing object replication
+<minio-replication-behavior-existing-objects>`. Specifically, resynchronization
+only applies to those replication configuration rules created where
+:mc-cmd:`~mc replicate add` includes ``"existing-objects"``.
+
+Use :mc-cmd:`mc replicate ls` to list the replication rules for a bucket and
+verify which rules have existing object replication enabled.
+
+Considerations
+--------------
+
+Resynchronization Requires Time
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Resynchronization is a background processes that continually checks objects in
+the source MinIO bucket and copies them to the remote as-needed. The time
+required for replication to complete may vary depending on the number and size
+of objects, the throughput to the remote MinIO deployment, and the load on the
+source MinIO deployment. Total time for completion is generally not predictable
+due to these variables.
+
+MinIO recommends configuring load balancers or proxies to direct traffic only
+to the healthy cluster until synchronization completes. The following commands
+can provide insight into the resynchronization status:
+
+- :mc-cmd:`mc replicate status` on the source and remote to track total 
+  replicated data.
+
+- Run ``mc ls -r --versions ALIAS/BUCKET | wc -l`` against both source and
+  remote to validate the total number of objects and object versions on each.
+
+Replication of Locked Objects
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+MinIO supports replicating objects held under 
+:ref:`WORM Locking <minio-object-locking>`. Both replication buckets *must* have
+object locking enabled for MinIO to replicate the locked object. For
+active-active configuration, MinIO recommends using the *same* 
+retention rules on both buckets to ensure consistent behavior across
+sites.
+
+You must enable object locking during bucket creation as per S3 behavior. 
+You can then configure object retention rules at any time.
+Object locking requires :ref:`versioning <minio-bucket-versioning>` and
+enables the feature implicitly.
+
+Resynchronize Objects after Data Loss
+-------------------------------------
```

Comment:
Note: Removed the references to re-creating users. It's cumbersome and covered as part of the pre-requisites.